qa/integration-tests/src/test/java/io/zeebe/broker/it/workflow/CancelWorkflowInstanceTest.java

```diff
@@ -127,4 +127,39 @@ public void shouldFailToCompleteJobAfterCancel() {
     assertJobCanceled();
     assertWorkflowInstanceCanceled("process");
   }
+
+  @Test
+  public void shouldNotCancelJob() {
+    // given
+    clientRule
+        .getClient()
+        .newCreateInstanceCommand()
+        .bpmnProcessId("process")
+        .latestVersion()
+        .send()
+        .join();
+
+    final List<ActivatedJob> jobEvents = new ArrayList<>();
```

Comment:
We don't need jobs here. Just use the `RecordingExporter` and get any `WorkflowInstanceRecord`.